frontend_lib/test/component/CardPopup.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow, configure } from 'enzyme'
+import CardPopup from '../../src/component/CardPopup/CardPopup.jsx'
+require('../../src/component/CardPopup/CardPopup.styl')
+
+describe('<CardPopup />', () => {
+  const props = {
+    customClass: 'randomCustomClass',
+    customHeaderClass: 'RandomCustomHeaderClass',
+    customColor: 'yellow',
+    onClose: () => {},
+    hideCloseBtn: false,
+    customStyle: {
+      color: 'yellow'
+    },
+    children: '<div>I am a children of CardPopup</div>'
+  }
+
+  const wrapper = shallow(
+    <CardPopup
+      customClass={props.customClass}
+      customHeaderClass={props.customHeaderClass}
+      customColor={props.customColor}
+      onClose={props.onClose}
+      hideCloseBtn={props.hideCloseBtn}
+      customStyle={props.customStyle}
+      children={props.children}
+    />
+  )
+
+  it(`should have the class "${props.customClass}"`, () =>
+    expect(wrapper.find(`.${props.customClass}`)).to.have.lengthOf(1)
+  )
+
+  it(`should have the class "${props.customHeaderClass}"`, () =>
```

Comment:
be more specific about the tests. Don't just check that the class exists in the component, also check that it is on the right DOM element.